.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,44 @@
+name: Fix code styling
+
+on:
+  workflow_dispatch:
```

Comment:
Should this also not run on commit to the master branch as well?

Reply:
I would probably stick to a dispatch, to mimic 
https://github.com/laravel/.github/blob/main/.github/workflows/coding-standards.yml

Reply:
I mean, just the lint (not the fix)

Reply:
I see; in any case, if lint should format for workflow_dispatch and lint on push, we need 2 different workflows files.
I propose to merge this one, and I'll make a PR for the linting part on push into master

WDYT ?

Reply:
Can we have a lint workflow for PRs as well please?